client/public/README.md

```diff
@@ -0,0 +1,13 @@
+# Luigi Client
+
+## Overview
+
+Luigi is a micro front-end orchestration framework written in Svelte. Use Luigi Core to implement the core application, and Luigi Client to ensure communication between the application and the micro front-end it hosts. 
+
+For details on Luigi Client, see [this](https://github.com/kyma-project/luigi/tree/master/client) document.
+
+For details on Luigi Core, see [this](https://github.com/kyma-project/luigi/tree/master/core) document.
+
+If you want to try Luigi out, see the [examples](https://github.com/kyma-project/luigi/tree/master/core/examples).
+
+For documentation on Luigi Core, see [Luigi documentation](https://github.com/kyma-project/luigi/tree/master/docs).
```

Comment:
Maybe we can change it to:

`For more documentation on Luigi framework, see [Luigi documentation](https://github.com/kyma-project/luigi/tree/master/docs). ` 

because it is not only about Luigi Core. 🤔